README.md

```diff
@@ -31,14 +37,112 @@ docker run -p 8090:8090 -d \
     -e MANTA_TIMEOUT=4000 \
     -e MANTA_METRIC_REPORTER_MODE=JMX \
     -e MANTA_HTTP_RETRIES=3 \
+    -e JETTY_SERVER_PORT=8090 \
     joyent/manta-monitor
 ```
 
-The parameter MANTA_HTTP_RETRIES, above defines the number of times to retry
+* HTTPS mode (secured):
```

Comment:
Try to not exceed 80 characters per line in markdown files and code. Exceptions are totally ok, but aim to get them close to that length.